whatsapp.py

```diff
@@ -107,7 +107,7 @@ def count_questions(df, regex=True):
 
 def count_curses(df):
     counter = pd.DataFrame()
-    curse_list = ["פאק", "שיט", "סעמק", "זונה", "דמט", "דאם", "דאמ", "לעזאזל", "זין", "רבאק"]
+    curse_list = ["פאק","פאקינג", "שיט", "סעמק","כוסאמק", "זונה", "דמט", "דאם", "דאמ", "לעזאזל", "זין", "רבאק"]
     for curse in curse_list:
```

Comment:
What are you doing here? why not generate the df directly from the list?

Reply:
this is the part that adds the + after each letter in the final regex. It would be quicker to add them manually, but I thought this ways the curses will be more readable (and also that way I maintain the original curse for the table

Reply:
OK, pls open an issue for me to review the logic here :)